src/components/InstallFeedback.js

```diff
@@ -4,222 +4,281 @@ import {
+                    width: 60%;
+                    border-radius: 4px;
+                    border: 1px solid;
+                  `}
+                />
+              </div>
+              <div>
+                <p>{t('installFeedback.email')}</p>
+                <input
+                  value={userEmail}
+                  placeholder="datanerd@example.com"
+                  onChange={(e) => {
+                    setUserEmail(e.target.value);
+                    setShouldSubmit(
+                      isValidEmail(e.target.value) ||
+                        e.target.value.length === 0
+                    );
+                  }}
+                  css={css`
+                    padding: 0.5rem;
+                    border-radius: 4px;
+                    border: 1px solid;
+                    margin-right: auto;
+                    width: 30%;
+                  `}
+                />
 
-  const callToActionText =
-    feedbackType === 'Positive'
-      ? `Any suggestions for how to make it even better?`
-      : `We'd love to hear what went wrong so we can fix it.`;
+                {userEmail && !isValidEmail(userEmail) && (
+                  <p
+                    css={css`
+                      // using && here to increase specificity over the p styling in the section above
+                      && {
+                        font-size: 1rem;
+                        margin: auto auto auto 0;
+                        color: var(--attention-notification-critical);
+                      }
+                    `}
+                  >
+                    {t('installFeedback.validEmail')}
+                  </p>
+                )}
+              </div>
+              <div
+                css={css`
+                  margin-top: -1rem;
+                  font-size: 0.75rem;
+                  color: var(--secondary-text-color);
+                  width: 50%;
+                `}
+              >
+                {t('installFeedback.emailDisclaimer')}
+              </div>
 
-  return (
-    <Surface
-      base={Surface.BASE.PRIMARY}
-      css={css`
-        padding: 2rem;
-        margin-bottom: 2rem;
-        display: flex;
-        flex-direction: row;
-        align-items: center;
-        border: 1px solid ${feedbackStyles.backgroundColor};
-        background-color: ${feedbackStyles.backgroundColor};
-      `}
-    >
-      <Icon
-        size="50px"
-        name={`fe-thumbs${feedbackType === 'Positive' ? 'up' : 'down'}`}
-        css={css`
-          color: #ffffff;
-        `}
-      />
-      <div
-        css={css`
-          display: flex;
-          flex-direction: column;
-          align-items: flex-start;
-          margin-left: 2rem;
-          margin-right: 2rem;
-        `}
-      >
-        <h2
-          css={css`
-            color: ${feedbackStyles.headingTextColor};
-            margin-bottom: 0;
-          `}
-        >
-          {feedbackText}
-        </h2>
+              <div css={css``}>
```

Comment:
got an empty css prop here
```suggestion
              <div>
```